cmd/norduser/main.go

```diff
@@ -25,7 +27,7 @@ func main() {
 		log.Fatalf("Error on listening to UNIX domain socket: %s\n", err)
```

Comment:
How about using limiting listener?
```
// limit count of requests on socket at the same time from
// non-authorized users to prevent from crashing daemon
listener = netutil.LimitListener(listener, 100)
```